rtloader/common/builtins/datadog_agent.c

```diff
@@ -103,6 +106,11 @@ void _set_get_clustername_cb(cb_get_clustername_t cb)
     cb_get_clustername = cb;
 }
 
+void _set_send_log_cb(cb_send_log_t cb)
+{
+    cb_send_log = cb;
```

Comment:
Is there a defined contract/doc that ensures this is only called before we start sending logs?  Otherwise this doesn't look thread safe (I realize this is not specific to this function at all, but still worth checking IMO)

Reply:
After a 2nd pass it seems that this is only meant to be called from the thread owning the python interpreter, so in between calls to the Python C API?

Reply:
Such checks occur in the first file of the PR `pkg/collector/python/datadog_agent.go`.